src/components/JosekiAdmin/JosekiAdmin.tsx

```diff
@@ -407,9 +407,11 @@ export class JosekiAdmin extends React.PureComponent<JosekiAdminProps, JosekiAdm
                     ]}
                 />
                 <div className="explorer-stats">
-                    <span>Page visits: {this.state.page_visits || "..."}</span>
```

Comment:
You probably want to use interpolate here, 

`interpolate(_("Page visits: {{count}}"), {"count": this.state.page_visits || "..."})` 

that way translators can move that count around for grammatical correctness. Sometimes they come back and want us to use things like `ngettext` singular/plural changes too, but hopefully in this case it doesn't matter/make sense?

Reply:
Will look into it.